app.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics.controllers");
+
+const app = express();
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+const handlePsqlErrors = (err, req, res, next) => {
```

Comment:
Please remove code that is not currently being used by your server.  It is not needed yet so simply shouldn't be here.